From 8939c73cb386f7f301d0423b048590409da6d2ee Mon Sep 17 00:00:00 2001
From: Rob Herring <r.herring@freescale.com>
Date: Tue, 19 Feb 2008 12:46:50 -0600
Subject: [PATCH 005/505] ENGR00065563 Remove udma_test_driver when SDMA is
 enabled

This patch removes the udma_test_driver from SDMA based platforms.

Signed-off-by: Rob Herring <r.herring@freescale.com>
---
 module_test/Makefile | 11 +++++------
 1 file changed, 5 insertions(+), 6 deletions(-)

diff --git a/module_test/Makefile b/module_test/Makefile
index 6007e43..1c58990 100644
--- a/module_test/Makefile
+++ b/module_test/Makefile
@@ -1,11 +1,10 @@
 
 ifneq ($(KERNELRELEASE),)
-# kbuild part of makefile 
+# kbuild part of makefile
 
-# When adding modules, either add to the 'module_list' for all platfoms or add 
+# When adding modules, either add to the 'module_list' for all platfoms or add
 # below to the 'module_list +=...' list for each platform that needs it.
-module_list-y := mxc_rtic_test.o \
-		mxc_udma_testdriver.o 
+module_list-y := mxc_rtic_test.o
 
 module_list-$(CONFIG_MXC_PMIC_MC13783) += mxc_pm_test.o mxc_pmic_power_testmod.o
 module_list-$(CONFIG_MXC_IPC) += ipctestmod.o
@@ -45,12 +44,12 @@ EXTRA_CFLAGS := -DLINUX_KERNEL
 
 ## NOTE: The following should be removed. They are directly requesting kernel
 ##       source.
-EXTRA_CFLAGS += -I$(LINUXPATH)/drivers/mxc/security/rng/include -I$(LINUXPATH)/drivers/mxc/security/sahara2/include 
+EXTRA_CFLAGS += -I$(LINUXPATH)/drivers/mxc/security/rng/include -I$(LINUXPATH)/drivers/mxc/security/sahara2/include
 
 else
 
 # Need to remove sahara objects because the test app uses same objects
-all: 
+all:
 	$(MAKE) -C $(LINUXPATH) M=`pwd` KBUILD_OUTPUT=$(KBUILD_OUTPUT) ARCH=arm CROSS_COMPILE=$(CROSS_COMPILE)  modules
 
 install:
-- 
1.8.0

